docs/nodes/compute/troubleshooting.md

```diff
@@ -174,3 +177,52 @@ When using IPv6 on a node, the network is unreachable.
 4. **Test Network Connectivity:**
 
     - Use `ping6` to ping the local IPv6 gateway or known IPv6 addresses like Google's DNS `2001:4860:4860::8888` to test connectivity.
+
+## 5) Persistent Storage Corruption
+### Issue Summary
+
+During operations, if the persistent storage test reveals data corruption, the system may be unable to properly read or validate data stored in the persistent volumes, potentially resulting in errors or data loss.
+
+#### Symptoms
+
+- Errors indicating that the data in persistent volumes is not valid JSON.
+- Persistent storage read failures logged during system checks.
```

Comment:
- Errors in the `persistent_storage` field from the diagnostic on the index page of a CRN or on the `/status/check/fastapi` API endpoint.
- The endpoint `/state/increment` on the diagnostic VM returns an error 500.
- The field `diagnostic_vm_latency` is missing from the metrics.